src/components/spinner/spinner.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Spinner from './spinner';
+import notes from './notes.md';
+
+storiesOf('Spinner', module).add('default', () => {
+  const as = select('as', OptionsHelper.colors, OptionsHelper.colors[3]);
```

Comment:
Can we take the default value from the component? I.e. `Spinner.defaultProps.as`